Dockerfile.linux.imagebuilder

```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20180117125404
+
+WORKDIR /
```

Comment:
I don't think we should be coping the contents into the root folder.  I think a subfolder would be more appropriate (should align with Dockerfile.linux.testrunner)